.github/pull_request_assignment.yml

```diff
@@ -103,108 +95,24 @@
     paths:
       - "specification/network/**"
     reviewers:
-      - JackTn
-      - keryul
+      - rkmanda
     branches:
       - release-network-Microsoft.Network-official
 
 - rule:
     paths: "specification/sql/**"
     reviewers:
-      - v-xuto
+      - rkmanda
     branches:
       - dev-sql-
 
 - rule:
     paths: "specification/@(financialdataservices|standbypool)/**"
     reviewers:
```

Comment:
is this for both control and data plane specs?

Reply:
Similar for this rule I suspect it will never be triggered.

Reply:
I see it in the private repo `RPSaaSDev`:

- https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSDev/specification/financialdataservices/resource-manager
- https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSDev/specification/standbypool/resource-manager

Reply:
But these are only mgmt again so probably can me removed as they will essentially never be reached.

Reply:
ok lets remove if redundant

Reply:
Same here.